### PR TITLE
Improve graphing asethetic

### PR DESCRIPTION
**Pull request by Alek99**
### Example (from just bases styles applied hooked up to the theme)
image

## Todo 
- add color_scheme prop
- Clean up the tooltip in dark mode for some graphs: funnel and radar

## Example code to test

"""Welcome to Reflex! This file outlines the steps to create a basic app."""

import reflex as rx

```python
data = [
    {"name": "Page A", "uv": 4000, "pv": 2400, "amt": 2400},
    {"name": "Page B", "uv": 3000, "pv": 1398, "amt": 2210},
    {"name": "Page C", "uv": 2000, "pv": 9800, "amt": 2290},
    {"name": "Page D", "uv": 2780, "pv": 3908, "amt": 2000},
    {"name": "Page E", "uv": 1890, "pv": 4800, "amt": 2181},
    {"name": "Page F", "uv": 2390, "pv": 3800, "amt": 2500},
    {"name": "Page G", "uv": 3490, "pv": 4300, "amt": 2100},
]

data01 = [
    {"x": 100, "y": 200, "z": 200},
    {"x": 120, "y": 100, "z": 260},
    {"x": 170, "y": 300, "z": 400},
    {"x": 170, "y": 250, "z": 280},
    {"x": 150, "y": 400, "z": 500},
    {"x": 110, "y": 280, "z": 200},
]

dataf = [
    {"value": 100, "name": "Sent", "fill": rx.color("accent", 5)},
    {"value": 80, "name": "Viewed", "fill": rx.color("accent", 6)},
    {"value": 50, "name": "Clicked", "fill": rx.color("accent", 7)},
    {"value": 40, "name": "Add to Cart", "fill": rx.color("accent", 8)},
    {"value": 26, "name": "Purchased", "fill": rx.color("accent", 9)},
]

datad = [
    {
        "subject": "Math",
        "A": 20,
        "B": 110,
        "fullMark": 150,
    },
    {
        "subject": "Chinese",
        "A": 38,
        "B": 130,
        "fullMark": 150,
    },
    {
        "subject": "English",
        "A": 86,
        "B": 130,
        "fullMark": 150,
    },
    {
        "subject": "Geography",
        "A": 99,
        "B": 100,
        "fullMark": 150,
    },
    {
        "subject": "Physics",
        "A": 55,
        "B": 90,
        "fullMark": 150,
    },
    {
        "subject": "History",
        "A": 65,
        "B": 85,
        "fullMark": 150,
    },
]


datae = [
    {
        "x": 45,
        "y": 100,
        "z": 150,
        "errorY": [30, 20],
        "errorX": 5,
    },
    {
        "x": 100,
        "y": 200,
        "z": 200,
        "errorY": [20, 30],
        "errorX": 3,
    },
    {
        "x": 120,
        "y": 100,
        "z": 260,
        "errorY": 20,
        "errorX": [5, 3],
    },
    {
        "x": 170,
        "y": 300,
        "z": 400,
        "errorY": [15, 18],
        "errorX": 4,
    },
    {
        "x": 140,
        "y": 250,
        "z": 280,
        "errorY": 23,
        "errorX": [6, 7],
    },
    {
        "x": 150,
        "y": 400,
        "z": 500,
        "errorY": [21, 10],
        "errorX": 4,
    },
    {
        "x": 110,
        "y": 280,
        "z": 200,
        "errorY": 21,
        "errorX": [5, 6],
    },
]
class State(rx.State):
    """The app state."""
 
    ...  
      
  
def index() -> rx.Component:
    # Welcome Page (Index)
    return rx.container(
    rx.recharts.area_chart(
        rx.recharts.area(
            data_key="uv",
            type_="monotone",
        ),   
        rx.recharts.brush(
            data_key="uv", height=30
        ), 
        rx.recharts.graphing_tooltip(),
        rx.recharts.x_axis(data_key="name"),
        rx.recharts.y_axis(),
        data=data,
        width="100%",
        height=250,
    ),      
    rx.recharts.bar_chart( 
        rx.recharts.graphing_tooltip(),
        rx.recharts.bar(
            data_key="uv",
        ),  
        rx.recharts.x_axis(data_key="name"),
        rx.recharts.y_axis(),
        data=data,
        width="100%",
        height=250, 
    ),          
    rx.recharts.line_chart( 
        rx.recharts.graphing_tooltip(),
        rx.recharts.x_axis(data_key="name"),
        rx.recharts.y_axis(),
        rx.recharts.line(
            data_key="uv",
        ),
        rx.recharts.line(
            data_key="pv",
        ),        
        data=data,
        width="100%",
        height=300,
    ),
    rx.recharts.scatter_chart(
        rx.recharts.graphing_tooltip(),
        rx.recharts.scatter(
            data=data01,
        ),
        rx.recharts.x_axis(data_key="x", type_="number"),
        rx.recharts.y_axis(data_key="y"),
        width="100%",
        height=300,
    ),
    rx.recharts.composed_chart( 
        rx.recharts.bar(
            data_key="amt", bar_size=20
        ),
        rx.recharts.cartesian_grid(
            stroke_dasharray="3 3",
            vertical=False,
        ),
        rx.recharts.graphing_tooltip(),
        rx.recharts.area(
            data_key="uv"
        ),  
        rx.recharts.line(
            data_key="pv", 
            type_="monotone",
        ),   
        
        rx.recharts.x_axis(data_key="name"),
        data=data, 
        height=250,  
        width="100%", 
    ),        
    rx.recharts.composed_chart( 
        rx.recharts.bar(
            data_key="amt", bar_size=20
        ),
        rx.recharts.cartesian_grid( 
            stroke_dasharray="3 3",
        ),
        rx.recharts.graphing_tooltip(),
        rx.recharts.area(
            data_key="uv"
        ),  
        rx.recharts.line(
            data_key="pv", 
            type_="monotone",
        ),   
        rx.recharts.x_axis(data_key="name"),
        rx.recharts.y_axis(),
        data=data, 
        height=250,  
        width="100%",  
    ),
    rx.recharts.funnel_chart(
        rx.recharts.funnel(
            rx.recharts.label_list(
                position="right",
                data_key="name",
            ),
            data_key="value",
            data=dataf,  
        ),  
        rx.recharts.graphing_tooltip(),
        rx.recharts.legend(),
        width="100%",
        height=300,
    ), 
    rx.recharts.scatter_chart(
        rx.recharts.scatter(
            rx.recharts.error_bar(
                data_key="errorY",
                direction="y",
                width=4,
                stroke_width=2, 
            ),
            rx.recharts.error_bar(
                data_key="errorX",
                direction="x",
                width=4,
                stroke_width=2,
            ),
            data=datae,
            name="A",
        ),
        rx.recharts.x_axis(
            data_key="x", name="x", type_="number"
        ),
        rx.recharts.y_axis(
            data_key="y", name="y", type_="number"
        ),
        width="100%",
        height=300,
    ),     
    rx.recharts.radar_chart(
        rx.recharts.radar(
            data_key="A",
        ),
        rx.recharts.radar(
            data_key="B",
        ), 
        rx.recharts.polar_grid(),
        rx.recharts.polar_angle_axis(data_key="subject"),
        rx.recharts.graphing_tooltip(),
        data=datad, 
        width="100%",
        height=300,
    ), 
    rx.recharts.pie_chart(
        rx.recharts.pie( 
            data=dataf,
            data_key="value",
            label=True,
        ),
        rx.recharts.graphing_tooltip(),
        rx.recharts.legend(),
        width="100%",
        height=300,
    ), 
    rx.button(         
        "Dark", 
        on_click=rx.toggle_color_mode 
    ), 
    width="100%",   
    )               
               
app = rx.App(    
    theme=rx.theme( 
        appearance="light",
        has_background=True,
        radius="large",
        accent_color="mint", 
        gray_color="mauve" 
    ) 
) 
app.add_page(index)

### CODE REVIEW

**Comment by masenf** [comment]
This looks really nice 😎

File: reflex/components/recharts/cartesian.py [hunk 1]
```diff
@@ -378,16 +392,22 @@ class Line(Cartesian):
...
+        "stroke": Var.create_safe(Color("accent", 10)),
+        "fill": Var.create_safe(Color("accent", 4)),
+    }
 
     # The dot is shown when user enter an area chart and this chart has tooltip. If false set, no active dot will not be drawn. If true set, active dot will be drawn which have the props calculated internally.
-    active_dot: Var[bool]
+    active_dot: Var[bool] = {
+        "stroke": Var.create_safe(Color("accent", 2)),
+        "fill": Var.create_safe(Color("accent", 10)),
+    }
```
Comment (masenf) [inline reflex/components/recharts/cartesian.py:410]: that annotation on these doesn't seem quite right anymore

File: reflex/components/recharts/polar.py [hunk 2]
```diff
@@ -70,11 +71,11 @@ class Pie(Recharts):
...
 
-    # fill color
-    fill: Var[str]
+    # Stoke color
+    stroke: Var[Union[str, Color]] = Var.create_safe(Color("accent", 9))
 
-    # stroke color
-    stroke: Var[str]
+    # Fill color
+    fill: Var[str] = Var.create_safe(Color("accent", 3))
```
Comment (masenf) [inline reflex/components/recharts/polar.py:78]:
```suggestion
    fill: Var[Union[str, Color]] = Var.create_safe(Color("accent", 3))
```

there's another one like this below too

File: reflex/components/recharts/cartesian.py [hunk 3]
```diff
@@ -304,6 +316,9 @@ class Area(Cartesian):
     # Valid children components
     _valid_children: List[str] = ["LabelList"]
 
+    #
```
Comment (masenf) [inline reflex/components/recharts/cartesian.py:319]:
```suggestion
    # The color scheme of the Area chart
```

**Review by picklelo — COMMENTED** [review]
Should the fill / stroke colors be consistent?

File: reflex/components/recharts/cartesian.py [hunk 4]
```diff
@@ -284,22 +291,25 @@ class Area(Cartesian):
     alias = "RechartsArea"
 
     # The color of the line stroke.
-    stroke: Var[Union[str, Color]]
+    stroke: Var[Union[str, Color]] = Var.create_safe(Color("accent", 9))
```
[the remark this answers is not in the document]
Comment (Alek99) [reply reflex/components/recharts/cartesian.py:294]: Keeping the value for the default display in docs

**Review by masenf — APPROVED** [review]

**Review by picklelo — APPROVED** [review]

**Pull request merged**